pkg/kubelet/types/labels.go

```diff
@@ -17,10 +17,20 @@ limitations under the License.
 package types
 
 const (
-	KubernetesPodNameLabel       = "io.kubernetes.pod.name"
-	KubernetesPodNamespaceLabel  = "io.kubernetes.pod.namespace"
-	KubernetesPodUIDLabel        = "io.kubernetes.pod.uid"
-	KubernetesContainerNameLabel = "io.kubernetes.container.name"
+	KubernetesPodNameLabel       = "k8s.io/pod-name"
```

Comment:
I think for all these labels there should be a fallback to the old labels otherwise this would be an breaking change and wouldn't work with any pods that are previously spawned before this patch.